### PR TITLE
BETA: Register @p2.is-a.dev

### PR DESCRIPTION
**Pull request by code-hunt-in**
Added `@p2.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/ap2.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "code-hunt-in",
+        "email": "aishwary.pandore@somaiya.edu"
+    },
+    "record": {
+        "TXT": "aishwary pandore"
```
Comment (Stef-00012) [inline domains/ap2.json:7]: TXT records alone do nothing

File: domains/app.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "<code-hunt-in>",
+    "email": "<aishwary.pandore@somaiya.edu>"
+  }
+  "record": {
+    "TXT": "Aishwary Pandore"
```
Comment (Stef-00012) [inline domains/app.json:7]: TXT records alone do nothing

File: domains/app.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "<code-hunt-in>",
+    "email": "<aishwary.pandore@somaiya.edu>"
```
Comment (Stef-00012) [inline domains/app.json:4]: Username and email shouldn't not have the `<>`

File: domains/@p2.json
Comment (Stef-00012) [inline domains/@p2.json:1]: File name non not contain `@`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by code-hunt-in — COMMENTED** [review]
Thank you for guidance

File: domains/@p2.json
Comment (Stef-00012) [inline domains/@p2.json:1]: File name non not contain `@`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/@p2.json
Comment (code-hunt-in) [reply domains/@p2.json:1]: is aish ok?

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**